api/src/middleware/index.ts

```diff
@@ -10,9 +10,7 @@ const isWhitelistedOrigin = (origin: string) =>
   origin.includes("metabase.com") ||
   origin.includes(FRONTEND_URL)
 
-export function setupMiddleware(app: Express) {
-  const isHosted = !!VERCEL_ENV
```

Comment:
this is moved up because it's set on the `app` but I need to pass `router` instead